feature-libs/cart/main/core/facade/active-cart.service.ts

```diff
@@ -287,20 +287,18 @@ export class ActiveCartService implements ActiveCartFacade, OnDestroy {
   requireLoadedCart(forGuestMerge = false): Observable<Cart> {
     // For guest cart merge we want to filter guest cart in the whole stream
     // We have to wait with load/create/addEntry after guest cart will be deleted.
-    const cartSelector$ = (
-      forGuestMerge
-        ? this.cartEntity$.pipe(
-            filter(() => !Boolean(getLastValueSync(this.isGuestCart())))
-          )
-        : this.cartEntity$
-    ).pipe(filter((cartState) => !cartState.loading));
+    const cartSelector$ = forGuestMerge
+      ? this.cartEntity$.pipe(
+          filter(() => !Boolean(getLastValueSync(this.isGuestCart())))
+        )
+      : this.cartEntity$;
```

Comment:
I think we need this filter `filter((cartState) => !cartState.loading)`. When cart is loading, it should not be created. Without the filter, the state is this:
![Screen Shot 2022-01-06 at 4 37 51 PM](https://user-images.githubusercontent.com/44440575/148448768-62924930-ffc3-4c47-ad17-61a9fa76dfb3.png)

With the filter, the state is this:
![Screen Shot 2022-01-06 at 4 38 58 PM](https://user-images.githubusercontent.com/44440575/148448797-20248d8b-ab55-4485-9d59-59d57b973d0b.png)

Reply:
Should we simply take the fix from #14747 ?

Reply:
yes, we can take the fix from #14747